### ISSUE TITLE
Suggestion: Add command to create the `Amx Mod X` 

### ISSUE THREAD

**Issue by xpt1x** [evandrocoan/AmxxEditor#5]
You should use a Menu itself not submenu from packages 
It allows users quickly to start a **New Plugin**
You should add **Edit Plugins.ini** too and maybe get its path from `include_directory` which is already defined in `Amxmodx.SublimeSettings` by trimming it a lil bit 😊

What you think @evandrocoan  ?

![oldstyle](https://user-images.githubusercontent.com/33655587/38452259-0eda57c4-3a5e-11e8-9753-00d9d36ef443.jpg)

**Comment by evandrocoan** [comment]
I could add a command to create the menu and detect the `plugins.ini`.

I already include a full menu when installing the https://github.com/evandrocoan/AmxxChannel, currently you can add the menu by creating this file on you `Packages/User/Main.sublime-menu` folder of Sublime Text:
```js
[
    {
        "caption": "Amx Mod X",
        "id": "amxmodx",
        "children":
        [
            {
                "command": "new_amxx_include",
                "caption": "New Include"
            },
            {
                "command": "new_amxx_plugin",
                "caption": "New Plug-In"
            },
            {
                "command": "open_file",
                "args": {"file": "${packages}/User/Amxmodx/AmxxPawn.sh"},
                "caption": "Configure Linux/Cygwin Compiler"
            },
            {
                "command": "open_file",
                "args": {"file": "${packages}/User/Amxmodx/AmxxPawn.bat"},
                "caption": "Configure Windows/Bat Compiler"
            },
            { "caption": "-" },
            {
                "command": "open_file",
                "args": { "file": "${packages}/User/AmxmodxConsole.sublime-settings" },
                "caption": "Edit Amxmodx Console Settings"
            },
            {
                "command": "open_file",
                "args": { "file": "${packages}/User/Amxmodx.sublime-settings" },
                "caption": "Edit Amxmodx Autocompletion Settings"
            },
        ]
    },
]
```

**Comment by janosharsanyi** [comment]
Why would you use the menu? @xpt1x  Shift+Ctrl+P New plugin
Can you imagine what would happen if every plugin put their menu there?

**Comment by xpt1x** [comment]
@janosharsanyi yeah i underatand that's why i highlighted plugins. Ini part coz i would really like to see this in New update i already added the menu for my convenience ( which was jst a normal suggestion as default amx studio too have this type ) but adding edit plugins. Ini in menu will be great, will you do that @evandrocoan ?

**Comment by evandrocoan** [comment]
Some day I may add this. But not this year at least.

**Comment by evandrocoan** [comment]
I just added a command to create the menu. But it will only be available when the version `3.0.0` of the package is released. I will release it soon as the following pull request gets merged:

1. https://github.com/wbond/package_control_channel/pull/7193 Renamed Amxmodx package to AmxxEditor

After that, you will be able to access the command on:

`Prefences -> Package Settings -> AmxxEditor -> Install AmxxEditor Menu`

![image](https://user-images.githubusercontent.com/5332158/42966490-aced05ae-8b73-11e8-90b6-41b10f6775c0.png)
